### PR TITLE
Include AWS driver usage & docker cli +x in documentation

### PR DESCRIPTION
**Pull request by kacole2**
Added chmod +x after using curl for the Docker CLI to make it
executable in the documentation.
Added an example for using the AWS EC2 driver in the documentation.

### CODE REVIEW

**Comment by nathanleclaire** [comment]
@kacole2 Please sign your commits (probably need something like `git commit -s --amend` and a force push).  Other than that this looks good to me

**Comment by kacole2** [comment]
ok. should be signed this time. thanks @nathanleclaire!

**Comment by nathanleclaire** [comment]
LGTM cc @ehazlett

**Comment by ehazlett** [comment]
LGTM

@moxiegirl PTAL

File: docs/index.md [hunk 1]
```diff
@@ -1063,6 +1064,12 @@ tcp://192.168.99.109:2376
 #### Amazon Web Services
 Create machines on [Amazon Web Services](http://aws.amazon.com).  You will need an Access Key ID, Secret Access Key and a VPC ID.  To find the VPC ID, login to the AWS console and go to Services -> VPC -> Your VPCs.  Select the one where you would like to launch the instance.
 
+Use `amazonec2` for the `--driver` and include the 3 required parameters. Depending upon the configured VPC, you may have to specify `--amazonec2-zone` if a subnet does not exist in Availability Zone `a`.
```
Comment (moxiegirl) [inline docs/index.md]: Thank you for the contribution.  Would you be willing to clean this section up a bit so it is easier to read.  I had one question -- I don't understand the reference to Availability Zone 'a' --- can you clarify.  Is Availability Zone 'a' the default and if it isn't in the default user needs to supply a parameter that points to the VPC's zone?

---> https://gist.github.com/moxiegirl/a7fe776b7c4127f97fd7

#### Amazon Web Services

To create machines on [Amazon Web Services](http://aws.amazon.com) (AWS), you must supply three required parameters:

* Access Key ID
* Secret Access Key
* VPC ID

You obtain these values from AWS.  To find the VPC ID:

!. Login to the AWS console 
2. Go to **Services -> VPC -> Your VPCs**.  
3. Select the VPC where you would like to launch the machine instance.

To create the machien instance, specify `--driver amazonec2` and the three required parameters. 

```
$ docker-machine create --driver amazonec2 --amazonec2-access-key AKI******* --amazonec2-secret-key 8T93C********* --amazonec2-vpc-id vpc-****** aws01
```

Depending upon the VPC you choose, you may also have to specify `--amazonec2-zone` if a subnet does not exist in **Availability Zone** `a`.

Comment (kacole2) [inline docs/index.md]: re VPC: The VPC id is in the ID column. for step 3, it should say: Use an ID specified in the ID column
re Availability Zone: `a` is used by default. To see if you have resources in availability zone `a`:
1.  Login to the AWS console
2.  Go to **Services -> VPC -> Subnets**
3.  Locate your selected VPC ID from the VPC column. Examine the `Availability Zone` column to see if `a` exists matching the VPC ID, such as `us-east1-a`. If `a` doesn't exist then you must specify `--amazonec2-zone` during creation such as `--amazonec2-zone c`. Or create a new subnet residing on Availability Zone `a`.

Comment (moxiegirl) [inline docs/index.md]: Thanks @kacole2 then, how about:

--> https://gist.github.com/moxiegirl/a7fe776b7c4127f97fd7

#### Amazon Web Services

To create machines on [Amazon Web Services](http://aws.amazon.com) (AWS), you must supply three required parameters:

* Access Key ID
* Secret Access Key
* VPC ID

You obtain these values from AWS.  To find the VPC ID:

1. Login to the AWS console 
2. Go to **Services -> VPC -> Your VPCs**.  
3. Locate the VPC ID you want from the **VPC** column. 
4. Examine the *Availability Zone** column to verify that zone `a` exists and matches the VPC ID.
  
    For example, `us-east1-a` is in the `a` availability zone. If the `a` zone is not present, you can create a new subnet in that zone or specify a different zone when you create the machine.

To create the machine instance, specify `--driver amazonec2` and the three required parameters. 

```
$ docker-machine create --driver amazonec2 --amazonec2-access-key AKI******* --amazonec2-secret-key 8T93C********* --amazonec2-vpc-id vpc-****** aws01
```

This example assumes the ID was found in the `a` availability zone. Use the `--amazonec2-zone` flag to specify a zone other than the `a` zone. For example, `--amazonec2-zone c` specifies the `c` zone.

**Comment by moxiegirl** [comment]
@kacole2 Thanks for the information and clarification. You can update your PR with what is in the GIST and rebase. If you don't want to do that work, let me know, we can close this and I'll carry the final step for you.  :-D

**Comment by kacole2** [comment]
that looks good. i'll update the PR

**Comment by kacole2** [comment]
forgot to sign it... coming once more

**Comment by kacole2** [comment]
signed and pulled. @moxiegirl please let me know if there is anything else to do. thanks

**Comment by moxiegirl** [comment]
Excellent @kacole2 LGTM.

**Comment by kacole2** [comment]
@moxiegirl rebased. let me know if it's good now. thanks!

**Comment by kacole2** [comment]
@moxiegirl why would the build fail? but i rebased it so it should be good

**Comment by nathanleclaire** [comment]
@kacole2 I noticed one of your commits was not signed, additionally I would suggest rebasing against `origin/master` (I see some duplicate / out of order commits in the PR) and squashing your commits into one (e.g. http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html).

**Comment by nathanleclaire** [comment]
This is the unsigned commit: https://github.com/kacole2/machine/commit/c0a7b9b6db69f613ac5f1085f05b2bd29200e8c9

**Comment by kacole2** [comment]
I somehow got way to deep down the rat hole and there are conflicts in files I never touched. Should I just close this out and open up a new PR?

**Comment by moxiegirl** [comment]
@kacole2 That might be easier for you. Reference this one in your new PR.  Thank you.

**Comment by dmp42** [comment]
Closing since this is carried through in another PR.

**Pull request closed without merging**